src/js/themes/base.js

```diff
@@ -617,10 +617,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // adjacent: {
         //   color: undefined,
         // },
-        // hover: {
-        //   background: undefined,
-        //   color: undefined,
-        // },
+        hover: {
+          background: 'active-background',
```

Comment:
Just confirming this is because we're removing the concept of "active" and now we want to enforce it here?

For best semantic use of tokens, can you also include a change to grommet-theme-hpe in the PR you plan to file which adjust this to:
```
hover: {
   background: 'background-hover',
}
```

Reply:
> Just confirming this is because we're removing the concept of "active" and now we want to enforce it here?

Yes

> For best semantic use of tokens, can you also include a change to grommet-theme-hpe

Yes, I'll plan to make that change

Reply:
Just checked, looks like we already have `calendar.day.hover.background` set to 'background-hover' in the hpe theme